CHANGELOG.md

```diff
@@ -1,3 +1,61 @@
+* **css:** add CSS vars to alerts ([#1424](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1424)) ([71e7bca](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/71e7bcaad7eb4ff91918df91fe5b198d798f2652))
+* **css:** add CSS vars to handle `.nav-tabs-light` and `.tab-content` ([#1427](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1427)) ([6ca8d1a](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/6ca8d1a9419fad360d2f755864cfa9797cd493db))
+* **css:** add CSS vars to navbars ([#1422](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1422)) ([59655c9](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/59655c9679f5b814a30effaef69ab31a6feba800))
+* **css:** define `.btn-no-outline`, `.btn-link` and `.btn-social` with CSS vars ([#1426](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1426)) ([20f1e5f](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/20f1e5f303f11e18f2c965fa58ae71e6e969ab0d))
+* **css:** improve `<mark>` rendering for a11y ([#1506](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1506)) ([a1d25dc](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/a1d25dc64e14bcdd31d046e08d21f3d8ecdf0116))
+* **list group:** add dark variant ([#1525](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1525)) ([19bcc40](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/19bcc4053a268cc8045dbbf49494d0344a1932e0))
+* **pagination:** add dark variant ([#1433](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1433)) ([3fb9a93](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/3fb9a938c44507e4696a2ec0373d6d49af9b82b2))
+* **pagination:** new `--#{$prefix}pagination-padding-end` CSS var ([#1416](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1416)) ([5a4f368](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/5a4f3688a33a9da3efc498c96bf5c1806df039d8))
+* **stepped process:** add dark variant ([#1434](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1434)) ([57633bb](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/57633bbcf90e6fe17e7b6503f67221eab4f4421e))
+* **utilities:** revamp `.img-thumbnail` and add desc in docs ([#1441](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1441)) ([344ab7e](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/344ab7ed5b8e31700a8eab3958ddfb069fbbb858))
+* **workflow:** temporarily deactivate BrowserStack ([#1547](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1547)) ([9dff4a1](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/9dff4a1a6836ab79aca7dc48944b14767039974f))
+
+
+
 # [5.2.0](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/compare/v5.1.3...v5.2.0) (2022-07-27)
 
 
```

Comment:
Weird display below, need to tackle anything in here ? (out of PR)

Reply:
Nice catch. It is because of the `<select>` in the message that is interpretated! I'm fixing it.